includes/function.login-header.php

```diff
@@ -40,7 +40,7 @@ function login_header( $title = 'Log In', $message = '', $wp_error = null ) {
 	$login_title = get_bloginfo( 'name', 'display' );
 
 	/* translators: Login screen title. 1: Login screen name, 2: Network or site name */
-	$login_title = sprintf( __( '%1$s &lsaquo; %2$s &#8212; WordPress' ), $title, $login_title );
+	$login_title = sprintf( __( '%1$s &lsaquo; %2$s &#8212; WordPress', 'two-factor' ), $title, $login_title );
```

Comment:
This file is currently being synced with the [same file in WordPress core](https://github.com/WordPress/WordPress/blob/60670888486617c6eeba7f6d038dd821941bbda3/wp-login.php#L25-L253) which is why we're not adding the custom textdomain for these strings. WordPress should pull those translations from the core translations.